parm/config/gfs/config.waveawipsgridded

```diff
@@ -9,5 +9,6 @@ echo "BEGIN: config.waveawipsgridded"
 . $EXPDIR/config.resources waveawipsgridded
 
 export DBNROOT=/dev/null
-
+export GEMPAK_GRIDS=${GEMPAK_GRIDS:-ak_10m at_10m ep_10m wc_10m glo_30m}
+# export GEMPAK_GRIDS=${GEMPAK_GRIDS:-ao_9km at_10m ep_10m wc_10m glo_30m}
```

Comment:
@JessicaMeixner-NOAA @AntonMFernando-NOAA I do not see GRIB2 templates for `ak_10m` under `parm/wave`.  This will cause problems for the `JGLOBAL_WAVE_PRDGEN_GRIDDED` JJob.  Instead, I see the `ao_9km` grid.  Do the GRIB2 templates need to be updated to include Alaska or should this list be reverted to `ao_9km at_10m ep_10m wc_10m glo_30m`?